components/report-manager-service/worker/worker.go

```diff
@@ -249,6 +250,21 @@ func (t *GenerateReportTask) Run(ctx context.Context, task cereal.Task) (interfa
 		}
 	}
 
+	config := lifecycle.NewConfiguration()
+	config.Rules = []lifecycle.Rule{
+		{
+			ID:     "expire-bucket",
+			Status: "Enabled",
+			Expiration: lifecycle.Expiration{
+				Days: 1,
+			},
+		},
+	}
+
+	err = t.ObjStoreClient.SetBucketLifecycle(context.Background(), job.RequestToProcess.RequestorId, config)
```

Comment:
Here we are setting life cycles rules always, we should set only if the bucket does not have the policy. So please check the existing policy and apply only if the bucket doesn't have any expiration policy.